packages/streamline/src/components/list-items/list-item-value/list-item-value.types.ts

```diff
@@ -2,4 +2,5 @@ import { ListItemProps } from '../list-item/list-item.types';
 
 export type ListItemValueProps = ListItemProps & {
   value: string;
+  isSelected?: boolean;
```

Comment:
To remove no?

Reply:
For me it make sens to have this props in the type, we have to know if the card is selected or not